rules/cloud/m365/audit/microsoft365_bypass_conditional_access.yml

```
@@ -0,0 +1,30 @@
+title: Azure Login Bypassing Conditional Access Policies
+id: 13f2d3f5-6497-44a7-bf5f-dc13ffafe5dc
+status: test
+description: |
+    Identifies a successful login to the Microsoft Intune Company Portal which could allow
+    bypassing Conditional Access Policies and InTune device trust using a tool like TokenSmith
+author: Josh Nickels, Marius Rothenbücher
+references:
+    - https://labs.jumpsec.com/tokensmith-bypassing-intune-compliant-device-conditional-access/
+    - https://github.com/JumpsecLabs/TokenSmith
+date: 2025-01-08
+logsource:
```

```suggestion
date: 2025-01-08
tags:
    - attack.defense-evasion
    - attack.t1078
logsource:
```
we use to put it here